julia/src/Launcher/load_params.jl

```diff
@@ -0,0 +1,67 @@
+using Random
+using Distributions
+
+function read_params(json, rng::AbstractRNG)
+  constant_kernel_param = json["transmission_probabilities"]["constant"]  |> float
+  household_kernel_param = json["transmission_probabilities"]["household"] |> float
+  hospital_kernel_param = get(json["transmission_probabilities"], "hospital", 0.0) |> float
+  friendship_kernel_param = get(json["transmission_probabilities"], "friendship", 0.0) |> float
+
+  mild_detection_prob = json["detection_mild_proba"]  |> float
+
+  tracking_prob = json["contact_tracking"]["probability"]  |> float
+  tracking_backward_delay = json["contact_tracking"]["backward_detection_delay"]  |> float
+  tracking_forward_delay = json["contact_tracking"]["forward_detection_delay"]  |> float
+  testing_time = json["contact_tracking"]["testing_time"]  |> float
+
+  phone_tracking_usage = json["phone_tracking"]["usage"] |> float
```

Comment:
- please use default here (0)
```suggestion
  phone_tracking_usage = get(json["phone_tracking"], "usage", 0.0) |> float
```
- `phone_tracking_testing_delay` also should have a default value

Reply:
changed it to the following: either `phone_tracking` is provided entirely or should not be provided at all